sources/FeatureDefinition.php

```diff
@@ -25,7 +25,7 @@ public function getName(): string
         return $this->name;
     }
 
-    public function getDescription()
+    public function getDescription(): string
```

Comment:
Again, I do not see how adding a return typehint on this public method can fix any deprecation.

I am curious to know where the deprecation is actually reported.